storage/helpers/customDatabaseRemover.go

```diff
@@ -0,0 +1,95 @@
+package helpers
```

Comment:
please rename `helpers` to `databaseRemover`

Reply:
done